examples/wiznet5k_simpleserver.py

```diff
@@ -31,6 +31,8 @@
 while True:
     conn, addr = server.accept()  # Wait for a connection from a client.
```

Comment:
Ok, if we're going to rewrite this..Move this outside the `while True`

Reply:
sorry, "we" means Adafruit? :) 
Anyway I see, what you mean. if you do pls rewirte code including below recv() issue.

Reply:
no, it means the pr writer. please fix